downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/SBMLDumper.java

```diff
@@ -0,0 +1,300 @@
+package org.reactome.release.downloadDirectory;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+
+import javax.xml.XMLConstants;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.Source;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.InvalidAttributeException;
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+public class SBMLDumper {
+	// Initialize data structures used in module
+	private static HashMap<String,HashMap<String,ArrayList<String>>> reactions = new HashMap<String,HashMap<String,ArrayList<String>>>();
+	private static HashMap<String,String> compartments = new HashMap<String,String>();
+	private static HashMap<String,HashMap<String,String>> species = new HashMap<String,HashMap<String,String>>();
+	
+	// Retrieve all human ReactionlikeEvents
+	public static void execute(MySQLAdaptor dba) throws Exception {
+		GKInstance human = dba.fetchInstance(48887L);
+		Collection<GKInstance> reactionInstances = dba.fetchInstanceByAttribute(ReactomeJavaConstants.ReactionlikeEvent, ReactomeJavaConstants.species, "=", human);
+		int count = 0;
+		for (GKInstance reactionInst : reactionInstances) {
+			dissectEvent(reactionInst);
+			count++;
+			if (count == 50) {
+				break;
+			}
```

Comment:
Just wondering why we stop after 50 instances?

Reply:
I think that SBML file generation is being done by EBI now, so this class is probably not even needed anymore. @cookersjs can confirm this.

Reply:
Yea that might be an artifact from when I was working on it. I'm going to delete this and the 'fetchCuratedComplex' classes in the near future

Reply:
:+1: